src/components/ui/FlowWindowContainer.tsx

```
@@ -0,0 +1,30 @@
+    const toggleVisibility = () => {
+        setIsVisible(!isVisible);
+    };
+
+    // Function to hide window
+    const hideWindow = () => {
+        setIsVisible(false);
+    };
+
+    // Expose the toggle function through a global handler
+    if (typeof window !== 'undefined') {
+        (window as any).toggleFlowWindow = toggleVisibility;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)